### PR TITLE
Dev ops slideupdates

### PR DESCRIPTION
**Pull request by rmbielby**
A few minor updates to the DevOps slide pack after the last workshop:
* Improvements to slide animations
* Brought in slide on the differences between GitHub and Azure DevOps

### CODE REVIEW

File: slides/workshopslides_gitDevOps.html
Comment (jen-machin) [inline slides/workshopslides_gitDevOps.html:1]: There's a typo on line 362 (sorry) - should be Kanban. Same on 612.

File: slides/workshopslides_gitDevOps.html [hunk 1]
```diff
@@ -475,52 +411,151 @@ <h1 data-id="quarto-animate-title">What are the key blockers to you using git?</
...
+</ul>
+</div>
+<div class="fragment fade-up">
 <ul>
 <li>A repository (or repo) is simply a folder containing all the version controlled files of a given project.</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>It can have multiple sub-folders that will can also be tracked.</li>
```
Comment (jen-machin) [inline slides/workshopslides_gitDevOps.html:444]: I think this is only meant to be "will be tracked" or "can be tracked"

File: slides/workshopslides_gitDevOps.html [hunk 2]
```diff
@@ -475,52 +411,151 @@ <h1 data-id="quarto-animate-title">What are the key blockers to you using git?</
...
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Undoing changes;</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Maintaining parallel variants of code.</li>
```
Comment (jen-machin) [inline slides/workshopslides_gitDevOps.html:429]: Could maybe add an extra bullet about safely storing your code somewhere central/"in the cloud"

File: slides/workshopslides_gitDevOps.html [hunk 3]
```diff
@@ -475,52 +411,151 @@ <h1 data-id="quarto-animate-title">What are the key blockers to you using git?</
...
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Create and view different branches (variants) of the code;</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Management tools such as Issues logs (GitHub) or Kambam board (Dev Ops)</li>
```
Comment (jen-machin) [inline slides/workshopslides_gitDevOps.html:476]: Kanban again (sorry) and there's a ; missing from the end of this line of text

File: slides/workshopslides_gitDevOps.html [hunk 4]
```diff
@@ -475,52 +411,151 @@ <h1 data-id="quarto-animate-title">What are the key blockers to you using git?</
...
+<ul>
 <li>Git records all the version control information within a hidden folder called .git</li>
 </ul>
+</div>
 </section>
 
 <section id="and-github-and-dev-ops" class="title-slide slide level1 center">
 <h1>And GitHub and Dev Ops…</h1>
+<div class="fragment fade-up">
 <p>GitHub and Dev Ops provide a space to store and interact with a repository.</p>
```
Comment (jen-machin) [inline slides/workshopslides_gitDevOps.html:457]: I'm pretty sure you say it out loud but some people in the workshops still get confused about git as the overarching "entity" vs GitHub/DevOps being a way to interact with that entity. People think git == GitHub and then they panic that their code/data is going to go somewhere public if they use git.

File: slides/workshopslides_gitDevOps.html [hunk 5]
```diff
@@ -475,52 +411,151 @@ <h1 data-id="quarto-animate-title">What are the key blockers to you using git?</
...
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Can try different methodologies in a properly controlled way</li>
 </ul>
+</div>
 </section>
 
 <section id="what-does-interacting-with-git-look-like" class="title-slide slide level1 center">
```
Comment (jen-machin) [inline slides/workshopslides_gitDevOps.html:561]: Is there an argument to add git stash to the list maybe? I don't know how much other people use it but we've definitely used it in the workshops before

**Review by jen-machin — COMMENTED** [review]
Slides look really good and everything seems to be working fine when I ran through them 😄 I have been a bit pedantic about typos, sorry. Also feel free to ignore all of my other suggestions as you see fit 🤣

File: slides/workshopslides_gitDevOps.html (hunk 2 above)
Comment (rmbielby) [reply slides/workshopslides_gitDevOps.html:429]: Have added a bit more focus on this in reference to Azure DevOps instead.

File: slides/workshopslides_gitDevOps.html (hunk 4 above)
Comment (rmbielby) [reply slides/workshopslides_gitDevOps.html:457]: Good point. Have added this explicitly in the slides now. Or at least I have locally, will push changes up once I've got internet again!

File: slides/workshopslides_gitDevOps.html (hunk 5 above)
Comment (rmbielby) [reply slides/workshopslides_gitDevOps.html:556]: I think I'm going to leave it out of this one, but have it on the list of things to add for the intermediate git workshop.

File: slides/workshopslides_gitDevOps.html [hunk 6]
```diff
@@ -475,52 +411,146 @@ <h1 data-id="quarto-animate-title">What are the key blockers to you using git?</
...
 <li>Edit your code (although there is some basic text editing functionality you can use).</li>
 </ul>
+</div>
+</section>
+
+<section id="whats-the-difference-between-git-github-and-azure-dev-ops" class="title-slide slide level1 center">
+<h1>What’s the difference between git, GitHub and Azure Dev Ops?</h1>
+<div class="fragment fade-up">
+<ul>
+<li>Git is the underlying software that you, GitHub and Azure DevOps use, providing the commands to record changes, create branches, merge branches, undo chages, etc.</li>
```
Comment (jen-machin) [inline slides/workshopslides_gitDevOps.html:509]: Sorry!!! One final typo (chages = changes)

**Review by jen-machin — COMMENTED** [review]
Just one more typo, but everything else looks good!!

**Pull request merged**